docker-compose.yml

```diff
@@ -1,18 +1,23 @@
-version: '2'
+version: '2.1'
```

Comment:
Ok - looks like we are running Docker Enginer `1.12.3`, which will work.

https://travis-ci.org/yarpc/yarpc-go/jobs/198210764#L304